Code/SeleniumBasics/UnitTest1.cs

```diff
@@ -0,0 +1,48 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using static System.Net.WebRequestMethods;
+
+namespace SeleniumBasics
+{
+    public class SelemiumBasicsEpamPageTests
```

Comment:
please do not forget to rename class in solution explorer as well :)

Reply:
done